eng/packages/http-client-csharp-mgmt/generator/Azure.Generator.Management/src/Providers/ResourceCollectionClientProvider.cs

```diff
@@ -163,6 +164,7 @@ private ConstructorProvider BuildResourceIdentifierConstructor()
             var bodyStatements = new List<MethodBodyStatement>();
 
             bodyStatements.Add(thisCollection.TryGetApiVersion(_resourceTypeExpression, $"{ResourceName}ApiVersion".ToVariableName(), out var apiVersion).Terminate());
+            bodyStatements.Add(apiVersion.Assign(new BinaryOperatorExpression("??", apiVersion, Literal(ManagementClientGenerator.Instance.InputLibrary.DefaultApiVersion))).Terminate());
```

Comment:
we could just do this:
```suggestion
            bodyStatements.Add(apiVersion.Assign(Literal(ManagementClientGenerator.Instance.InputLibrary.DefaultApiVersion), nullCollapse: true).Terminate());
```